src/components/ha-check-list-item.ts

```diff
@@ -6,6 +6,19 @@ import { customElement } from "lit/decorators";
 
 @customElement("ha-check-list-item")
 export class HaCheckListItem extends CheckListItemBase {
+  protected firstUpdated(): void {
+    super.firstUpdated();
+
+    if (document.dir === "rtl") {
+      this.updateComplete.then(() => {
+        const style = document.createElement("style");
+        style.innerHTML =
+          "span.mdc-deprecated-list-item__graphic { margin-left: var(--mdc-list-item-graphic-margin, 16px) !important; margin-right: 0px !important;}";
+        this.shadowRoot!.appendChild(style);
+      });
+    }
```

Comment:
why not just add this to the `styles` getting, like the override below for `--mdc-theme-secondary`?

Reply:
Done